### ISSUE TITLE
Auto & Cut Trans don't work...

### ISSUE THREAD

**Issue by tepaze** [danielbuechele/atemOSC#21]
Hi,

I'm using v2.4.4, and with Pure Data, Transition Auto and Cut don't work.
Many other thing work, some other don't (like aux bus assignment), but transition is very usefull....

I can write anything to correct it because i don't know Objective-C language, so i hope you can :-)

Best regard 

Tepaze

**Comment by sneat** [comment]
I don't know Pure Data, but perhaps it is sending a value that atemOSC can't handle. If possible, have it send an integer of `1` as the value and see how that goes.

This is likely the reason that a lot of the endpoints aren't working for you. There's a handy app called [OSCTestApp](http://vdmx.vidvox.net/blog/freebies) which you can use to send/test every single possible value at an address. This will help you track down exactly what's going wrong for you.

**Comment by tepaze** [comment]
Thank for reply,

I try to add integer value and it's work for some function (don't remember those, but i can found)
I'm gone to test the app and see... 
Hope i find solution :-)
Another question, how i can find previous version of atemOSC, there's no release...
Have a nice day.

**Comment by sneat** [comment]
If you are comfortable using git, you can checkout an older [commit](https://github.com/danielbuechele/atemOSC/commits/master) for the appropriate version.

Is there a reason that you're wanting a previous version?

**Comment by tepaze** [comment]
I want a previous version to use with previous ATEM version...
On a previous version, auto and cut work for me on an older version of ATEM (i don't remember whitch number)...

**Comment by sneat** [comment]
Ahh right. In that case you'll need to grab the appropriate commit hash from https://github.com/danielbuechele/atemOSC/commits/master

For instance, let's say you wanted to use the version before ATEM v6: find the commit prior to the one that adds v6 and copy the commit hash (c282339a21f2a863c33b8d4f587a7ba04ad2c97c in this instance for the commit titled "updated support for controlling DSKs").

If you already have a copy of the git repository you can skip this step:
```bash
git clone https://github.com/danielbuechele/atemOSC.git
```

then, to get that version:
```bash
cd atemOSC
git checkout c282339a21f2a863c33b8d4f587a7ba04ad2c97c
```

Your local folder will now be at the point in time that that commit was added. You can then run the `atemOSC` app in its previous version. If need be, grab a previous commit hash (SHA) until you get the version you need.

**Comment by tepaze** [comment]
Thank you :-)

**Comment by SteffeyDev** [comment]
Closing this because resolved, and opened #40 to address similar issues like this in the future.
